controllers/sriovnetwork_controller.go

```diff
@@ -173,8 +182,31 @@ func (r *SriovNetworkReconciler) Reconcile(ctx context.Context, req ctrl.Request
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *SriovNetworkReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	// Reconcile when the target namespace is created after the SriovNetwork object.
+	namespaceHandler := handler.Funcs{
+		CreateFunc: func(ctx context.Context, e event.CreateEvent, q workqueue.RateLimitingInterface) {
+			networkList := sriovnetworkv1.SriovNetworkList{}
+			err := r.List(ctx,
+				&networkList,
+				client.MatchingFields{"spec.networkNamespace": e.Object.GetName()},
+			)
+			if err != nil {
+				log.Log.WithName("SriovNetworkReconciler").
+					Info("Can't list SriovNetworks for namespace", "resource", e.Object.GetName(), "error", err)
+			}
+
+			for _, network := range networkList.Items {
```

Comment:
just a small improvement can you check if the namespace in the handler is equal to the namespace of the network from the list?

Reply:
Doesn't L191 ensure that? We are listing every SriovNetwork that has `spec.NetworkNamespace`  equal to the handler namespace.

Reply:
you are right!!
sorry I miss that one then all good